### DISCUSSION TITLE
KontroLIR - 100% Arduino compatible IR remote Control based on IRremote

### DISCUSSION THREAD

**Discussion by AnalysIR** [Arduino-IRremote/Arduino-IRremote#782] [Show and tell]
For anyone interested in a real remote control using IRremote, check out our blog post: [KontroLIR](https://www.analysir.com/blog/2019/07/06/kontrolir-the-first-arduino-compatible-ir-remote-control/)

Full details are provided in the blog post linked above. There is comprehensive documentation provided on our WiKI: [KontroLIR WiKi](https://wiki.analysir.com/index.php?title=KontroLIR_-_Arduino_compatible_IR_remote_control)

![KontroLIR_photo_top](https://user-images.githubusercontent.com/4930550/106495994-45086100-64b4-11eb-89c2-6fb1b1de9dc0.png)

KontroLIR now comes with integrated USB via the battery compartment and is programmed using the Arduino IDE.

**Comment by ajesquirell** [comment]
This is amazing!
